### PR TITLE
CAL-1416 Cleanup and remove the unused code of the old create event p…

### PR DESCRIPTION
**Pull request by nttuyen**
- remove old ui component for create/edit event and task

### CODE REVIEW

File: calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java [hunk 1]
```diff
@@ -454,4 +451,18 @@ public void createActionConfirms(UIConfirmation uiConfirmation) {
...
+
+  /**
+   * use in template
+   */
+  public int getTimeShift(){
+    int defaultTime = 2;
+    try {
+      defaultTime = Integer.parseInt(PropertyManager.getProperty("exo.calendar.default.event.suggest"));
+    } catch (NumberFormatException e) {
+      defaultTime = 1;
```
Comment (thomasdelhomenie) [inline calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java]: We should not swallow the exception and log an error to warn the administrator that the configuration is not correct.

Comment (nttuyen) [reply calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java]: ok let me fix this

Comment (nttuyen) [reply calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java]: @thomasdelhomenie , i'm done fixing log and increase test coverage rate

**Comment by trongtt** [comment]
@nttuyen could you also resolve conversation which has been fixed ?

**Comment by nttuyen** [comment]
@trongtt , yes done

**Comment by nttuyen** [comment]
@trongtt, ok, let me revert other files

**Comment by nttuyen** [comment]
@trongtt , i'm done revert resources bundle files, only edit _en.properties file

File: calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java [hunk 2]
```diff
@@ -454,4 +451,19 @@ public void createActionConfirms(UIConfirmation uiConfirmation) {
...
+  /**
+   * use in template
+   */
+  public int getTimeShift(){
+    int defaultTime = 2;
+    try {
+      defaultTime = Integer.parseInt(PropertyManager.getProperty("exo.calendar.default.event.suggest"));
+    } catch (NumberFormatException e) {
+      log.error("exo.calendar.default.event.suggest must be int, it currently is: {}", defaultTime);
+      defaultTime = 1;
```
Comment (trongtt) [inline calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java]: Why defaultTime is re-assigned to 1 ? which will mislead the error log message in the previous line.

File: calendar-webapp/src/main/resources/locale/portlet/calendar/CalendarPortlet_en.properties [hunk 3]
```diff
@@ -718,15 +677,15 @@ UISelectUserForm.label.Replace=Replace
...
-UIInvitationForm.action.Cancel=#{word.cancel}
+
+
+
+
+
+
+
+
+
```
Comment (trongtt) [inline calendar-webapp/src/main/resources/locale/portlet/calendar/CalendarPortlet_en.properties]: Too many empty lines ?

File: calendar-webapp/src/main/webapp/templates/calendar/webui/UICalendarPortlet.gtmpl [hunk 4]
```diff
@@ -8,7 +8,8 @@
   import org.exoplatform.calendar.webui.UICalendarPortletEditMode;
   import javax.portlet.MimeResponse;
   import javax.portlet.ResourceURL;
-  
+
+  def timeshift = uicomponent.getTimeShift();
```
Comment (trongtt) [inline calendar-webapp/src/main/webapp/templates/calendar/webui/UICalendarPortlet.gtmpl]: Should move this initializing variable to the place nearly its usage.

File: calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java (hunk 2 above)
Comment (nttuyen) [reply calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java]: i'm done fixing this

File: calendar-webapp/src/main/resources/locale/portlet/calendar/CalendarPortlet_en.properties (hunk 3 above)
Comment (nttuyen) [reply calendar-webapp/src/main/resources/locale/portlet/calendar/CalendarPortlet_en.properties]: i'm done fixing this

File: calendar-webapp/src/main/webapp/templates/calendar/webui/UICalendarPortlet.gtmpl (hunk 4 above)
Comment (nttuyen) [reply calendar-webapp/src/main/webapp/templates/calendar/webui/UICalendarPortlet.gtmpl]: i'm done fixing this

File: calendar-webapp/webpack.dev.js [hunk 5]
```diff
@@ -5,7 +5,7 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const apiMocker = require('connect-api-mocker');
 
 // change the server path to your server location path
-const exoServerPath = "../../../exo-servers/platform-5.2.x-create-event-SNAPSHOT/";
+const exoServerPath = "/home/vstorm83/workspace/exodev/PLF/packaging/platform-public-distributions/develop/plf-community-tomcat-standalone/target/platform-community-5.2.x-SNAPSHOT/platform-community-5.2.x-SNAPSHOT/";
```
Comment (trongtt) [inline calendar-webapp/webpack.dev.js]: Did you revert this change ?

[the remark this answers is not in the document]
Comment (trongtt) [reply calendar-webapp/webpack.dev.js]: Did you revert this change ?

[the remark this answers is not in the document]
Comment (nttuyen) [reply calendar-webapp/webpack.dev.js]: sorry i push this back

**Comment by nttuyen** [comment]
> @nttuyen The test coverage has been improved since code has been removed. I did a test, we can change it to 0.02 instead of 0.01. It is a small improvement but it is an improvement :)

@thomasdelhomenie , i've already done updating the test coverage rate
